frontend/src/components/UserSettings/UserNewsletter.vue

```diff
@@ -41,10 +41,6 @@ export default {
       this.$apollo
         .mutate({
           mutation: this.newsletterState ? subscribeNewsletter : unsubscribeNewsletter,
-          variables: {
-            email: this.$store.state.email,
```

Comment:
do we still need $store.state.email now?

Reply:
https://github.com/gradido/gradido/issues/2895